src/containers/couresCarousel/index.js

```diff
@@ -59,7 +59,7 @@ class CoursesCarousel extends Component {
           className={styles.navButton}
           type="button"
           onClick={this.prevItem}>
-          <IoIosArrowBack color="white" size="50px" />
+          <IoIosArrowBack color="white" className={styles.carouselArrows} />
         </button>
         <div className={styles.carousel}>
           <AliceCarousel
```

Comment:
Maybe `ref={el => this.Carousel = el}` needs to be replased by `ref={this.CarouselRef}`?

React.CreateRef() is intended for such goals like animation.